Ptt/AppDelegate.swift

```diff
@@ -32,6 +32,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             window.makeKeyAndVisible()
 
         }
+        let appearance = UINavigationBarAppearance()
+        appearance.configureWithDefaultBackground()
+        UINavigationBar.appearance().standardAppearance = appearance
+        UINavigationBar.appearance().compactAppearance = appearance
+        UINavigationBar.appearance().scrollEdgeAppearance = appearance
```

Comment:
Because the navigation bar color issue only happens on iOS 15 or later, I suggest adding `if #available(iOS 15, *)` to make this piece of code more precise. See: https://stackoverflow.com/a/69135729

Reply:
updated